app/Models/Instance/Cron.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace App\Models\Instance;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+
+class Cron extends Model
+{
+    use HasFactory;
+
+    /**
+     * The attributes that are mass assignable.
+     *
+     * @var array<string>
+     */
+    protected $fillable = [
+        'command',
+        'last_run',
```

Comment:
Technically all the dates in the DB finish with `_at` so ideally that would be `last_run_at`. But for this table, ok, it's not a product related table.

Reply:
I can do it, no prob